Libplanet/Store/IStore.cs

```diff
@@ -224,6 +224,29 @@ void PutBlock<T>(Block<T> block)
             IImmutableDictionary<Address, IValue> states
         );
 
+#pragma warning disable MEN002
+        /// <summary>
+        /// Looks up a state reference, which is a block's <see cref="Block{T}.Hash"/> that contains
+        /// an action mutating the <paramref name="address"/>' state.
+        /// </summary>
+        /// <param name="chainId">The chain ID to look up a state reference.</param>
+        /// <param name="address">The <see cref="Address"/> to look up.</param>
+        /// <param name="lookupUntil">The upper bound (i.e., the latest block) of the search range.
+        /// <see cref="Block{T}"/>s after <paramref name="lookupUntil"/> are ignored.</param>
+        /// <returns>Returns a nullable tuple consisting of <see cref="Block{T}.Hash"/> and
+        /// <see cref="Block{T}.Index"/> of the <see cref="Block{T}"/> with the state of the
+        /// address.</returns>
+        /// <typeparam name="T">An <see cref="IAction"/> class used with
+        /// <paramref name="lookupUntil"/>.</typeparam>
+        /// <seealso cref="IStore.StoreStateReference(Guid, IImmutableSet{Address}, HashDigest{SHA256}, long)"/>
+        /// <seealso cref="IStore.IterateStateReferences(Guid, Address, long?, long?, int?)"/>
+#pragma warning restore MEN002
+        Tuple<HashDigest<SHA256>, long> LookupStateReference<T>(
+            Guid chainId,
+            Address address,
+            Block<T> lookupUntil)
+            where T : IAction, new();
```

Comment:
@earlbread Although this is too late, there still is `StoreExtension.LookupStateReference<T>(this IStore store, Guid chainId, Address address, Block<T> lookupUntil) where T : IAction, new()`, and indeed the extension method is still used here and there.  Isn't it intended, right?  Should I remove `StoreExtension.LookupStateReference<T>()` method?

Reply:
It was my mistake. 😨 I thought that I removed `StoreExtension` class, but I didn't. We should remove `StoreExtension.LookupStateReference<T>()`.

Reply:
Ok, nevermind. It was just for being sure. I am going to remove it in my next patch.